library/network/src/modules/SuSEFirewall.rb

```diff
@@ -678,6 +678,15 @@ def IsVerbose
       Ops.greater_than(@verbose_level, 0)
     end
 
+    # Local function returns if protocol is supported by firewall.
+    # Protocol name must be in upper-cases.
+    #
+    # @param [String] protocol
+    # @return	[Boolean] if protocol is supported
```

Comment:
OK I will mention that to the commit message. As for the reverse dependencies, I believe it shouldn't be a major problem since querying for RPC services will simply return an empty list.